### PR TITLE
[AN-861] Minor improvement. Example walkthrough how to connect to the database.

### PR DESCRIPTION
**Pull request by alexey-temnikov**
improving doc based on the feedback - 
Establishing a connection manually via a URL could use a concrete example walkthrough when configuring an SSH tunnel.

### Summary
Minor Doc improvements

### Description
Added example how to connect to Neptune.

### Related Issue
https://bitquill.atlassian.net/browse/AN-861

### Additional Reviewers
@birschick-bq
@lyndonb-bq
@xiazcy
@simonz-bq
@alexey-temnikov
@kylepbit

### CODE REVIEW

File: markdown/sql.md [hunk 1]
```diff
@@ -33,6 +33,17 @@ The above properties are configurations that are shared across all query languag
 
 Each Builder method listed in the documentation is accepted as a property. For example, "**[port](https://tinkerpop.apache.org/javadocs/current/full/org/apache/tinkerpop/gremlin/driver/Cluster.Builder.html#port-int-)**(int port)" means that as a part of the connection string it may be configured as a property or added as a Key-Value Pair in Properties. See below for examples.
 
+#### Establishing Connection Manually
+ 
+Amazon Neptune Database is a VPC only service, and you need to establish an SSH tunnel before connecting to the database outside of the VPC (e.g. from the local machine).
+To establish an SSH tunnel, you need an EC2 instance deployed in the same VPC as Neptune Database. More information on how to configure SSH Tunnel is described in [Connecting to Neptune](/markdown/setup/configuration.md#using-an-ssh-tunnel-to-connect-to-amazon-neptune)
+
+- Step 1. Establish Port Forwarding through SSH Tunnel
```
Comment (lyndonbauto) [inline markdown/sql.md]: Nit: Use numbered list without step numbers instead of bullets.

File: markdown/sql.md [hunk 2]
```diff
@@ -33,6 +33,17 @@ The above properties are configurations that are shared across all query languag
...
+- Step 2. Add Neptune Database hostname in /etc/hosts (see [Adding SSH Tunnel Lookup](/markdown/setup/configuration.md#adding-a-ssh-tunnel-lookup) for more details).
+- Step 3. Enter Connection String. As an example in [DBVisualizer](https://www.dbvis.com/) enter *Database URL* `jdbc:neptune:sqlgremlin://<hostname>;port=8182;authScheme=NONE`. If you are using default port and using IAM athentication you do not need to supply addiitonal parameters.
```
Comment (kylepbit) [inline markdown/sql.md]: I think <hostname> and port here are tied to the SSH tunnel that you set up, is that correct? If so, can you note that?

File: markdown/sql.md [hunk 3]
```diff
@@ -33,6 +33,17 @@ The above properties are configurations that are shared across all query languag
 
 Each Builder method listed in the documentation is accepted as a property. For example, "**[port](https://tinkerpop.apache.org/javadocs/current/full/org/apache/tinkerpop/gremlin/driver/Cluster.Builder.html#port-int-)**(int port)" means that as a part of the connection string it may be configured as a property or added as a Key-Value Pair in Properties. See below for examples.
 
+#### Establishing Connection Manually
+ 
+Amazon Neptune Database is a VPC only service, and you need to establish an SSH tunnel before connecting to the database outside of the VPC (e.g. from the local machine).
+To establish an SSH tunnel, you need an EC2 instance deployed in the same VPC as Neptune Database. More information on how to configure SSH Tunnel is described in [Connecting to Neptune](/markdown/setup/configuration.md#using-an-ssh-tunnel-to-connect-to-amazon-neptune)
+
+- Step 1. Establish Port Forwarding through SSH Tunnel
+- Step 2. Add Neptune Database hostname in /etc/hosts (see [Adding SSH Tunnel Lookup](/markdown/setup/configuration.md#adding-a-ssh-tunnel-lookup) for more details).
```
Comment (lyndonbauto) [inline markdown/sql.md]: Windows doesn't have /etc/hosts. Should add windows and Mac.

Ultimately would like to remove this requirement asap

File: markdown/sql.md (hunk 1 above)
Comment (alexey-temnikov) [reply markdown/sql.md]: Changed

File: markdown/sql.md (hunk 3 above)
Comment (alexey-temnikov) [reply markdown/sql.md]: Rephrased. Will not be duplicting instructions.

File: markdown/sql.md (hunk 2 above)
Comment (alexey-temnikov) [reply markdown/sql.md]: Fixed

**Comment by alexey-temnikov** [comment]
Adjusted based on comments

**Comment by alexey-temnikov** [comment]
Adjusted based on feedback.

File: markdown/sql.md [hunk 4]
```diff
@@ -36,12 +36,12 @@ Each Builder method listed in the documentation is accepted as a property. For e
...
+To establish an SSH tunnel, you need an EC2 instance deployed in the same VPC as Neptune Database. More information on how to configure SSH Tunnel is described in [Connecting to Neptune](/markdown/setup/configuration.md#using-an-ssh-tunnel-to-connect-to-amazon-neptune).
 
-- Step 1. Establish Port Forwarding through SSH Tunnel
-- Step 2. Add Neptune Database hostname in /etc/hosts (see [Adding SSH Tunnel Lookup](/markdown/setup/configuration.md#adding-a-ssh-tunnel-lookup) for more details).
-- Step 3. Enter Connection String. As an example in [DBVisualizer](https://www.dbvis.com/) enter *Database URL* `jdbc:neptune:sqlgremlin://<hostname>;port=8182;authScheme=NONE`. If you are using default port and using IAM athentication you do not need to supply addiitonal parameters.
-- Step 4. Now you can connect to your Amazon Neptune Database instance.
+ 1. Establish Port Forwarding through SSH Tunnel
+2. Add Neptune Database hostname in `hosts` (see [Adding SSH Tunnel Lookup](/markdown/setup/configuration.md#adding-a-ssh-tunnel-lookup) for more details how to add configuraiton in Windows and MacOS).
```
Comment (kylepbit) [inline markdown/sql.md]:
```suggestion
2. Add Neptune Database hostname in `hosts` (see [Adding SSH Tunnel Lookup](/markdown/setup/configuration.md#adding-a-ssh-tunnel-lookup) for more details how to add configuration in Windows and macOS).
```

File: markdown/sql.md [hunk 5]
```diff
@@ -36,12 +36,12 @@ Each Builder method listed in the documentation is accepted as a property. For e
...
-To establish an SSH tunnel, you need an EC2 instance deployed in the same VPC as Neptune Database. More information on how to configure SSH Tunnel is described in [Connecting to Neptune](/markdown/setup/configuration.md#using-an-ssh-tunnel-to-connect-to-amazon-neptune)
+To establish an SSH tunnel, you need an EC2 instance deployed in the same VPC as Neptune Database. More information on how to configure SSH Tunnel is described in [Connecting to Neptune](/markdown/setup/configuration.md#using-an-ssh-tunnel-to-connect-to-amazon-neptune).
 
-- Step 1. Establish Port Forwarding through SSH Tunnel
-- Step 2. Add Neptune Database hostname in /etc/hosts (see [Adding SSH Tunnel Lookup](/markdown/setup/configuration.md#adding-a-ssh-tunnel-lookup) for more details).
-- Step 3. Enter Connection String. As an example in [DBVisualizer](https://www.dbvis.com/) enter *Database URL* `jdbc:neptune:sqlgremlin://<hostname>;port=8182;authScheme=NONE`. If you are using default port and using IAM athentication you do not need to supply addiitonal parameters.
-- Step 4. Now you can connect to your Amazon Neptune Database instance.
+ 1. Establish Port Forwarding through SSH Tunnel
```
Comment (kylepbit) [inline markdown/sql.md]:
```suggestion
1. Establish Port Forwarding through SSH Tunnel
```

File: markdown/sql.md [hunk 6]
```diff
@@ -36,12 +36,12 @@ Each Builder method listed in the documentation is accepted as a property. For e
...
+2. Add Neptune Database hostname in `hosts` (see [Adding SSH Tunnel Lookup](/markdown/setup/configuration.md#adding-a-ssh-tunnel-lookup) for more details how to add configuraiton in Windows and MacOS).
+3. Enter Connection String. As an example in [DBVisualizer](https://www.dbvis.com/) enter *Database URL* `jdbc:neptune:sqlgremlin://<hostname>;port=<port>;authScheme=NONE`. Default *port* is `8182`, however you can configure port forwarding to any available port on your machine. If you are using default port and using IAM athentication you do not need to supply addiitonal parameters.
```
Comment (kylepbit) [inline markdown/sql.md]:
```suggestion
3. Enter Connection String. As an example in [DBVisualizer](https://www.dbvis.com/) enter *Database URL* `jdbc:neptune:sqlgremlin://<hostname>;port=<port>;authScheme=NONE`. Default *port* is `8182`, however you can configure port forwarding to any available port on your machine. If you are using default port and using IAM authentication you do not need to supply additional parameters.
```

Comment (kylepbit) [reply markdown/sql.md]: What should <hostname> be, what should port be, based on the SSH tunnel that you've established?

File: markdown/sql.md [hunk 7]
```diff
@@ -33,6 +33,17 @@ The above properties are configurations that are shared across all query languag
...
+ 
+Amazon Neptune Database is a VPC only service, and you need to establish an SSH tunnel before connecting to the database outside of the VPC (e.g. from the local machine).
+To establish an SSH tunnel, you need an EC2 instance deployed in the same VPC as Neptune Database. More information on how to configure SSH Tunnel is described in [Connecting to Neptune](/markdown/setup/configuration.md#using-an-ssh-tunnel-to-connect-to-amazon-neptune).
+
+1. Establish Port Forwarding through SSH Tunnel
+2. Add Neptune Database hostname in `hosts` (see [Adding SSH Tunnel Lookup](/markdown/setup/configuration.md#adding-a-ssh-tunnel-lookup) for more details how to add configuration in Windows and macOS).
+3. Enter Connection String. As an example in [DBVisualizer](https://www.dbvis.com/) enter *Database URL* `jdbc:neptune:sqlgremlin://<hostname>;port=<port>;authScheme=NONE`. Default *port* is `8182`, however you can configure port forwarding to any available port on your machine. If you are using default port and using IAM authentication you do not need to supply additional parameters.
+4. Now you can connect to your Amazon Neptune Database instance.
+
+
```
Comment (lyndonbauto) [inline markdown/sql.md:57]: Nit:
```suggestion
```

**Review by lyndonbauto — APPROVED** [review]

**Pull request merged**